benchmarks/spectre_v1/CMakeLists.txt

```
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.5.1)
+set(CMAKE_BUILD_TYPE Release)
+
+project(safeside VERSION 0.1.0 LANGUAGES CXX)
+
+# Use C++11 without extensions
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_EXTENSIONS off)
+
+add_executable(unmitigated microbenchmark.cc)
+target_compile_options(unmitigated PUBLIC -O0)
+
+if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+	add_executable(slh_mitigated microbenchmark.cc)
+	target_compile_options(slh_mitigated PUBLIC -mspeculative-load-hardening -O0)
```

```suggestion
  add_executable(slh_mitigated microbenchmark.cc)
  target_compile_options(slh_mitigated PUBLIC -mspeculative-load-hardening -O0)
```

Build files in the repo indent with two spaces, or four in the case of continued lines.